### DISCUSSION TITLE
Running roc-recv on a embedded device using ALSA

### DISCUSSION THREAD

**Discussion by folibis** [roc-streaming/roc-toolkit#523] [Discuss]
I have a Linux embedded device and a speaker connected to it. I can play a wav file using `aplay file.wav` and I can list devices using:
```
aplay -L
default:CARD=am3517evm
    am3517evm, 
    Default Audio Device
```
Bun while running `roc-recv` as following I get errors:
roc-recv -vv --output=CARD=am3517evm --driver=alsa
00:01:04.531 [err] roc_sndio: no backend fround: driver=alsa inout=CARD=am3517evm
00:01:04.531 [err] roc_recv: can't open output file or device: driver=alsa output=CARD=am3517evm

I've tried:
```
--output=CARD=am3517evm,DEV=0
--output=CARD=0,DEV=0
--output=front:CARD=am3517evm,DEV=0
--output=front:CARD=0,DEV=0
--output=default
```

but nothing works, the error persists

what do I wrong?

**Comment by fallingcats** [comment]
This may not be relevant anymore but directly using `--driver alsa  --output default:CARD=am3517evm` should work

**Comment by gavv** [comment]
On fresh master it should be:

```
-o alsa://default:CARD=am3517evm
```

Please let me know if the issue is still relevant.
